inst/templates/BS4/navbar.html

```diff
@@ -16,11 +16,9 @@
       </ul>
       {{/left}}
 
-      {{#yaml}}{{#docsearch}}
       <form class="form-inline my-2 my-lg-0" role="search">
-        <input type="search" class="form-control mr-sm-2" aria-label="Search" name="search-input" id="search-input" placeholder="Search..." aria-label="Search for..." autocomplete="off">
+        <input type="search" class="form-control mr-sm-2" aria-label="Search" name="search-input" data-search-index="{{{search-index}}}" id="search-input" placeholder="Search..." aria-label="Search for..." autocomplete="off">
```

Comment:
data attribute for finding the search index whose location depends on the development mode.